src/objective-c/tests/PerfTests/PerfTests.m

```diff
@@ -359,7 +359,8 @@ - (void)testMultipleChannels {
   NSMutableArray<NSString *> *addrs = [NSMutableArray arrayWithCapacity:kNumAddrs];
   NSMutableArray<RMTTestService *> *services = [NSMutableArray arrayWithCapacity:kNumAddrs];
   for (int i = 0; i < kNumAddrs; ++i) {
-    addrs[i] = [NSString stringWithFormat:@"127.0.0.%d", (i + 1)];
+    // http://readme.localtest.me/
+    addrs[i] = [NSString stringWithFormat:@"%d.localtest.me", (i + 1)];
```

Comment:
a bit more details on why we need to switch to localtest.me

Reply:
In order to connect to local server using multiple channels, prior to this PR, In `src/objective-c/tests/run_one_test.sh` script, it attaches 127.0.0.x IP addresses to localhost and remove them after the test, which requires root permission
`sudo ifconfig lo0 alias 127.0.0.$i up`
`sudo ifconfig lo0 -alias 127.0.0.$i`

This change utilize the fact that `xxx.localtest.me` always resolve to 127.0.0.1, so we use different domain names to get multiple channels without the need of root permission and custom script run form bazel.

For details of localtest.me, check https://readme.localtest.me/